src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -320,7 +325,9 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
         });
 
         Handlebars.registerHelper('datetime', function(dateTime) {
-            return new Handlebars.SafeString(new Date(dateTime).toLocaleTimeString());
+            // round to the nearest minute
```

Comment:
Ah ok. Maybe the comment should just be adjusted to `// round up to the nearest minute` to be a bit clearer.

Providing an upper bound on the estimate makes sense to me, as its better to overshoot than undershoot. If the estimate is 12.0 minutes and that estimate gets rounded up in the edge case to 13 minutes, is that a big deal? Doesn't seem like it to me?